packages/databyss-editor/lib/clipboardUtils/deleteBlocksAtSelection.ts

```diff
@@ -124,6 +124,7 @@ export default (draft: EditorState) => {
   // collapse selection
   const _cursor = { offset: anchor.offset, index: anchor.index }
   draft.selection = {
+    $type: DocumentType.Selection,
```

Comment:
If we have to use `$type` or `DocumentType` outside of `@databyss-org/data` we've done something wrong. It is meant as a field internal to our database library for managing types. Let's discuss why we are using this here and some alternatives.